libs/langchain/langchain/memory/chat_message_histories/dynamodb.py

```diff
@@ -84,6 +93,6 @@ def clear(self) -> None:
         from botocore.exceptions import ClientError
 
         try:
-            self.table.delete_item(Key={"SessionId": self.session_id})
+            self.table.delete_item(self.key)
```

Comment:
`TypeError: delete_item() only accepts keyword arguments.`

IMHO this should be `self.table.delete_item(Key=self.key)`